aries-site/src/pages/components/checkboxgroup.js

```diff
@@ -0,0 +1,163 @@
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/checkboxgroup/CheckBoxGroupSimpleExample.js"
+          figma="https://www.figma.com/file/7Mm1xDBTOtPHqggEVpaD2N/HPE-Checkbox-Component?node-id=1745%3A43"
+          designer="https://designer.grommet.io/checkbox?id=HPE-design-system-hpedesignsystem-hpe-com"
+        >
+          <CheckBoxGroupSimpleExample />
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          CheckBoxGroup should be used to present a group of options to the
+          user. The user can select a single or multiple options.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Best Practices" level={3} gap="small">
+        <BulletedList
+          level={3}
+          items={[
+            `CheckBoxGroup should be used to select zero or more of options 
+              from a list.`,
+            'In a CheckBoxGroup, options are independent from one another.',
+            `Settings changed by checkboxes should not take effect until the 
+              user clicks to proceed.`,
+            `It is recommended to have only 10 or fewer options in a 
+              CheckBoxGroup. Use multiple CheckboxGroups if necessary.`,
+          ]}
+        />
+      </Subsection>
+      <Subsection name="Usage" level={3} gap="small">
+        <Text weight="bold">The dos for CheckBoxGroup:</Text>
+        <BulletedList
+          level={3}
+          items={[
+            `Always label CheckBoxGroups with short and concise labels.`,
+            `Give every CheckBox in the group a short, but descriptive 
+              value.`,
+            `Clearly separate Checkbox Groups from other groups on the same
+              page.`,
+            `Organize Checkboxes in a logical order by grouping related
+              options.`,
+            `Consider placing most common options first in the group.`,
+          ]}
+        />
+        <Text weight="bold">The don'ts for CheckBoxGroup:</Text>
+        <BulletedList
+          level={3}
+          items={[
+            `Don’t align checkboxes horizontally in a group.`,
+            `Refrain from using CheckboxGroup when only one item can be 
+              selected from a list.`,
+            `Organize checkboxes in a logical order by grouping related 
+              options.`,
+          ]}
+        />
+      </Subsection>
+      <Subsection name="Accessibility" level={3} gap="small">
+        <SubsectionText>
+          When using multiple checkboxes in a single FormField, use a
+          CheckBoxGroup. If you're using a single checkbox, use{' '}
+          <Anchor label="Checkbox" href={nameToPath('Checkbox')} />.
+        </SubsectionText>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Variants">
```

Comment:
I would add a variant that shows how the indeterminate `checkbox` is used in a group.

Reply:
Nice idea. I'll create one for that.

Reply:
Update: indeterminate hasn't been added into checkboxgroup yet, so let's add this example in later on.

https://github.com/grommet/grommet/pull/4033

Reply:
okay that sounds good thanks for letting me know!